### ISSUE TITLE
Client doesn't properly handle Dropdown

### ISSUE THREAD

**Issue by iagopiimenta** [docusign/docusign-ruby-client#17]
I'm getting this error while loading a template that contains listTabs:

`#<NoMethodError: undefined method `build_from_hash' for []:Array>`

![image](https://user-images.githubusercontent.com/5642707/54482930-cf486d00-4829-11e9-8634-a04471c05fb3.png)

Context of the `lib/docusign_esign/models/tabs.rb:430`:
```ruby
=> # def _deserialize(type, value)
=> type
"Array"
=> value
{:listItems=>[{:text=>"Sample TEXT", :value=>"SAMPLE VALUE", :selected=>"false"}, {:text=>"Other (Please specify)", :value=>"Other (Please specify)", :selected=>"false"}, {:text=>"Select", :value=>"Select", :selected=>"true"}], :value=>"Select", :width=>153, :shared=>"false", :required=>"false", :locked=>"false", :requireAll=>"false", :tabLabel=>"business_structure", :font=>"lucidaconsole", :bold=>"false", :italic=>"false", :underline=>"false", :fontColor=>"black", :fontSize=>"size10", :documentId=>"XXXXXX", :recipientId=>"XXXXXXXX", :pageNumber=>"3", :xPosition=>"42", :yPosition=>"360", :tabId=>"XXXXXXX", :templateLocked=>"false", :templateRequired=>"false"}
=> DocuSign_eSign::VERSION
"1.0.2"
```

**Comment by LarryKlugerDS** [comment]
Thank you for the problem report.

What API method were you using?
Did this happen before the request was sent to DocuSign or while the SDK was processing the response?

(You can tell by looking at the [API log](https://support.docusign.com/guides/ndse-user-guide-api-request-logging) from DocuSign. If the request/response is shown there then the problem happened when the SDK was parsing the response.)

**Comment by iagopiimenta** [comment]
The problem happened when the SDK was parsing the response. I just fetched a template: `DocuSign_eSign::TemplatesApi.new(client).get(account_id, template_id)`

**Comment by LarryKlugerDS** [comment]
Thank you for the bug report. I've filed internal ticket DCM-3192

**Comment by LarryKlugerDS** [comment]
Good news, this issue has been fixed with the 3.0.0.rc.1 release of the SDK and [gem](https://rubygems.org/gems/docusign_esign/versions/3.0.0.rc1). The source is available as branch [ngopala/v2.1-19.2.02](https://github.com/docusign/docusign-ruby-client/tree/ngopala/v2.1-19.2.02). 

It will be released to master and the gem will be updated in the next week.

Please comment on this issue if you spot any issues with the RC (release candidate) or the forthcoming release.

Thanks,
Larry
